src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -59,15 +75,17 @@ function lineData(node: any) {
       recentValues.forEach((val: any) => {
         if (isFinite(val)) {
           chData.push(val);
-          dsMax = Math.max(dsMax, val);
-          dsMin = Math.min(dsMin, val);
         }
       });
+      const localMax = Math.max(...chData);
+      node.data.dsMax = ((node.data.dsMax) ? Math.max(localMax, node.data.dsMax) : localMax);
+      const localMin = Math.min(...chData);
+      node.data.dsMin = ((node.data.dsMin) ? Math.min(localMin, node.data.dsMin) : localMin);
       dataset.data = chData;
       chartDataSets.push(dataset);
     }
   });
-  stepY = (dsMax - dsMin) / 2;
+  stepY = ((node.data.dsMax * scalar) - node.data.dsMin) / 2;
```

Comment:
Just double checking that the scalar should only apply to the max? Looking at the PT ticket, it's not clear to me... I see that LB says no to a global minimum, but the bullet points say the buttons increase/decrease the top and bottom.

Reply:
Originally, per Leslie's request she only wanted it to apply to the max. 
But based on our thread with Leslie this morning and your +/- zoom suggestion, I need to refactor this